{"links":{"static":"docs/api/validation/class/ValidationErrorsCustomAttribute/property/errors","html":"docs/api/validation/class/ValidationErrorsCustomAttribute/property/errors/index.html","fragment":"docs/api/validation/class/ValidationErrorsCustomAttribute/property/errors/index-fragment.html","self":"docs/api/validation/class/ValidationErrorsCustomAttribute/property/errors/index.json"},"name":"errors","kindString":"Property","flags":{"isExported":true},"sources":[{"fileName":"aurelia-validation.d.ts","line":438,"character":7}],"type":{"type":"array","elementType":{"type":"reference","name":"RenderedError","id":5448}}}